src/contrib/cluster/Akka.DistributedData.LightningDB/LmdbDurableStore.cs

```diff
@@ -0,0 +1,222 @@
+﻿#region copyright
+// -----------------------------------------------------------------------
+//  <copyright file="LmdbDurableStore.cs" company="Akka.NET project">
+//      Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2017 Akka.NET project <https://github.com/akkadotnet>
+//  </copyright>
+// -----------------------------------------------------------------------
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Text;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.DistributedData.Durable;
+using Akka.Event;
+using Akka.Serialization;
+using LightningDB;
+
+namespace Akka.DistributedData.LightningDB
```

Comment:
JVM version by default uses LMDB backend for storage. I think, it's better to move that dependency to a separate package however.